docs/deploying.md

```diff
@@ -369,6 +369,150 @@ support.
+
+2. Get the `docker-compose` binary.
+
+		$ sudo wget https://github.com/docker/compose/releases/download/1.2.0/docker-compose-`uname  -s`-`uname -m` -O /usr/local/bin/docker-compose
+
+	This command installs the binary in the `/usr/local/bin` directory. 
+	
+3. Add executable permissions to the binary.
+
+		$  sudo chmod +x /usr/local/bin/docker-compose
+		
+
+### Do some housekeeping
+
+1. Remove any previous images.
+
+		$ docker rmi -f $(docker images -q -a )
+		
+	 This step is a house keeping step. It prevents you from mistakenly picking up
+	 an old image as you work through this example.
+		
+2. Edit the `distribution/cmd/registry/config.yml` file and remove the `tls` block.
+
+	If you worked through the previous example, you'll have a `tls` block. To use
+	TLS with the registry and Nginx, you should terminate the TLS on the Nginx
+	rather than the registry.
+
+### Configure your system
+
+1. Change to the `distribution/contrib/basicauth` directory.
+
+	This directory contains a single `docker-compose.yml` configuration file and an `nginix`
+	configuration directory. The `nginx` directory contains:
+	
+	| File                      | Description                                                                        |
+  |---------------------------|------------------------------------------------------------------------------------|
+  | `Dockerfile`              | Describes the Nginx image with a configuration that supports basic authentication. |
+  | `docker-registry-v2.conf` | Specifies the headers required by the registry.                                    |
+  | `registry-basic.conf`     | Specifies the basic authentication configuration required by the registry.         |
+  | `registry.conf`           | Specifies the base configuration, including TLS, required by the registry.         |
+  | `ssl`                     | Contains pre-generated certificates to use in the example.                         |
+  | `test.pass`               | Contains a username-password combination to use in testing.                        |
+			
+2. Create `certs.d` directory on your local machine. 
+
+        $ sudo mkdir -p /etc/docker/certs.d/localhost:5441
+	  
+3. Copy the sample certification to your new `certs.d` directory.
+	
+        $ sudo cp nginx/ssl/registry-ca+ca.pem /etc/docker/certs.d/localhost:5441/ca.crt
+
+### Build and run
+
+1. Go up to the `distribution/basicauth` directory
+
+	This directory includes a single `docker-compose.yml` configuration.
+	
+        nginx:
+          build: "nginx"
+          ports:
+            - "5441:5441"
+          links:
+            - registryv2:registryv2
+        registryv2:
+          build: "../../"
+          ports:
+            - "5000"
+
+ This configuration builds a new `nginx` image as specified by the
+ `nginx/Dockerfile` file. The `registryv2` image is built from the
+ `distribution/Dockerfile` you've used previously.
+	
+3. Build `nginx` and the registry 2.0 image.
```

Comment:
Step "2" is missing (which shouldn't be a problem in the rendered Markdown output, but is a bit confusing reading the source)